.github/RELEASE_TEMPLATE.md

```diff
@@ -1,150 +1,95 @@
-| ---                                                     | ---                       | ---                                                          |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.sha256` | <pre>Linux 64-bit</pre>   | Checksum for the Linux executable.                           |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.linux64.tar.gz` | <pre>Linux 64-bit</pre>   | Linux executable with auto-completion script for bash shell. |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.macos64.sha256` | <pre>MacOs 64-bit</pre>   | Checksum for the MacOS executable.                           |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.macos64.tar.gz` | <pre>MacOs 64-bit</pre>   | MacOS executable with necessary system libraries.            |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.win64.sha256`   | <pre>Windows 64-bit</pre> | Checksum for Windows executable.                             |
-| `cardano-wallet-jormungandr@vYYYY-MM-DD.win64.zip`      | <pre>Windows 64-bit</pre> | Windows executable with necessary DLLs.                      |
+| Link                                                                                                                                        | Audience                                                   |
+| ---                                                                                                                                         | ---                                                        |
+| [API Documentation](https://input-output-hk.github.io/cardano-wallet/api/{{GIT_TAG}})                                                       | Users of the Cardano Wallet API                            |
+| CLI Manual: [jormungandr](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface) / [byron](https://github.com/input-output-hk/cardano-wallet/wiki/Wallet-Command-Line-Interface-(cardano-wallet-byron)) | Users of the Cardano Wallet API                            |
+| [Docker Manual](https://github.com/input-output-hk/cardano-wallet/wiki/Docker/11246e7633eba794fb90fab385239753ba32d70e)                     | Users of the Cardano Wallet API                            |
+| [Haddock Documentation](https://input-output-hk.github.io/cardano-wallet/haddock/{{GIT_TAG}})                                               | Haskell Developers using the `cardano-wallet` as a library |
 
-# Installation Instruction
+# Installation Instructions 
 
 <!-- Specific installation steps for this particular release. This should
 basically captures whatever is currently available on the repository at
-the moment of releasing. --> 
+the moment of releasing. -->
 
 ## Linux 64-bit
 
-1. Install [jormungandr@X.Y.Z](https://github.com/input-output-hk/jormungandr/releases/tag/vX.Y.Z) from the official repository.
+1. Install [jormungandr@{{JORM_TAG}}](https://github.com/input-output-hk/jormungandr/releases/tag/{{JORM_TAG}}) from the official repository.
```

Comment:
Perhaps we should also mention cardano-node here? 
```
1. Install jormungandr@VERSION or cardano-node@VERSION
2. Download respective cardano-wallet binary (jormungandr or cardano-node compatible)
```
... and maybe add similar info for all: Linux, Windows, MacOS and Docker...

Reply:
61fc85f74d41a73add840514096514f95e852522